lib/lightning/attempts/prom_ex_plugin.ex

```diff
@@ -20,4 +24,60 @@ defmodule Lightning.Attempts.PromExPlugin do
       ]
     )
   end
+
+  @impl true
+  def polling_metrics(_opts) do
+    stalled_attempt_threshold_seconds =
+      Application.get_env(:lightning, :metrics)[
+        :stalled_attempt_threshold_seconds
+      ]
+
+    Polling.build(
+      :lightning_attempt_polling_events,
+      5000,
+      {__MODULE__, :stalled_attempt_count, [stalled_attempt_threshold_seconds]},
+      [
+        last_value(
+          [:lightning, :attempt, :queue, :stalled, :count],
+          event_name: [:lightning, :attempt, :queue, :stalled],
+          description: "The count of attempts stuck in the `available` state",
+          measurement: :count
+        )
+      ]
+    )
+  end
+
+  def stalled_attempt_count(threshold_seconds) do
+    trigger_stalled_attempt_metric(Process.whereis(Repo), threshold_seconds)
+  end
+
+  defp trigger_stalled_attempt_metric(nil, _threshold_seconds) do
+    nil
+  end
+
+  defp trigger_stalled_attempt_metric(repo_pid, threshold_seconds) do
+    # NOTE: During local testing of server starts, having the pid was not enough
+    # a call to .get_state was also required, otherwise the metric triggers before
+    # the Repo GenServer is available.
+    #
+    :sys.get_state(repo_pid)
+
+    threshold_time =
+      DateTime.utc_now()
+      |> DateTime.add(-1 * threshold_seconds)
+
+    query =
+      from a in Attempt,
+        select: count(a.id),
+        where: a.state == :available,
+        where: a.inserted_at < ^threshold_time
+
+    count = Repo.one(query)
+
+    :telemetry.execute(
+      [:lightning, :attempt, :queue, :stalled],
```

Comment:
minor detail: to put the event in a constant (module attribute like `@event_name`) to make it clear that the one dispatched is the same of the publishing.